src/Acuminator/Acuminator.Utilities/Roslyn/Semantic/Dac/Enums/DacType.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Acuminator.Utilities.Roslyn.Semantic.Dac
+{
+	public enum DacType
+	{
+		None,
```

Comment:
Why do we need this value?

Reply:
To be similar to PXGraph + some a value to represent "not a DAC". I think may be it will be used in the future. Or maybe not. Let's remove it for now